### PR TITLE
Wdu 47 nearest vehicle scheduling and result ui

### PR DESCRIPTION
**Pull request by nlasagni**
* Implemented nearest vehicle scheduling and result ui

### CODE REVIEW

File: mobile/src/main/java/com/wedriveu/mobile/app/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import com.wedriveu.mobile.tripscheduling.view.SchedulingViewImpl;
+import com.wedriveu.mobile.tripscheduling.viewmodel.SchedulingViewModel;
+import com.wedriveu.mobile.tripscheduling.viewmodel.SchedulingViewModelImpl;
+import com.wedriveu.mobile.util.Constants;
+import com.wedriveu.mobile.util.location.LocationService;
+import com.wedriveu.mobile.util.location.LocationServiceImpl;
+
+/**
+ *
+ * @author Marco Balsassarri, Nicola Lasagni
```
Comment (mbaldassarri) [inline mobile/src/main/java/com/wedriveu/mobile/app/MainActivity.java]: is this javadoc syntax corrent? shouldn't we use 
@author Nicola Lasagni
@author Marco Baldassarri
in order to generate a correct javadoc file?

File: mobile/src/main/java/com/wedriveu/mobile/service/login/LoginService.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.wedriveu.mobile.service.login;
+
+/**
+ *
+ * This service interacts with the AuthenticationService backend to perform the user login.
+ * This service is part of the Model in MVVM pattern.
+ *
+ * @author  Marco Baldassarri, Nicola Lasagni
```
Comment (mbaldassarri) [inline mobile/src/main/java/com/wedriveu/mobile/service/login/LoginService.java]: same as MainActivity

File: mobile/src/main/java/com/wedriveu/mobile/app/ComponentFinder.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.wedriveu.mobile.app;
+
+import android.support.v4.app.Fragment;
+
+/**
+ *
+ * Component finder interface
+ *
+ * @author Marco Baldassarri, Nicola Lasagni
```
Comment (mbaldassarri) [inline mobile/src/main/java/com/wedriveu/mobile/app/ComponentFinder.java]: Same as MainActivity

**Comment by nlasagni** [comment]
Merged with commit [https://github.com/nlasagni/S3-16-we-drive-u/commit/a6b282bf1697827ea2c5581a2cf47afb1aa4d09b](url)

**Pull request closed without merging**